Assets/AltTester/Editor/Scripts/AltTestRunner.cs

```diff
@@ -88,11 +89,20 @@ public static void RunTests(TestRunMode testMode)
             }
             if (AltTesterEditorWindow.EditorConfiguration.createXMLReport)
             {
+                if (File.Exists(AltTesterEditorWindow.EditorConfiguration.xMLFilePath) || Directory.Exists(AltTesterEditorWindow.EditorConfiguration.xMLFilePath))
+                {
+                    FileAttributes attr = File.GetAttributes(AltTesterEditorWindow.EditorConfiguration.xMLFilePath);
+                    if (Path.GetExtension(AltTesterEditorWindow.EditorConfiguration.xMLFilePath) != ".xml" && !attr.HasFlag(FileAttributes.Directory))
+                        AltTesterEditorWindow.EditorConfiguration.xMLFilePath = "test-report.xml";
+                    else if (attr.HasFlag(FileAttributes.Directory))
+                        AltTesterEditorWindow.EditorConfiguration.xMLFilePath = AltTesterEditorWindow.EditorConfiguration.xMLFilePath + "\\" + "test-report.xml";
```

Comment:
will this `\\` work on a mac? Maybe there would be a method that we can use to create a path that is platform independent?

Reply:
@SandyEma9

Reply:
@rucindrea  was testing and working on Mac